packages/runtime/src/prop-controllers/descriptors.ts

```diff
@@ -554,26 +530,6 @@ export function Images(options: ImagesOptions = {}): ImagesDescriptor {
   return { type: Types.Images, version: 1, options }
 }
 
-export type LinkValue = Link
-
-export type LinkOptions = Options<{
-  preset?: LinkValue
-  label?: string
-  defaultValue?: Link
-  options?: { value: Link['type']; label: string }[]
-  hidden?: boolean
-}>
-
-export type LinkDescriptor<_T = LinkValue> = { type: typeof Types.Link; options: LinkOptions }
-
-/**
- * @deprecated Imports from `@makeswift/runtime/prop-controllers` are deprecated. Use
- * `@makeswift/runtime/controls` instead.
- */
-export function Link(options: LinkOptions = {}): LinkDescriptor {
-  return { type: Types.Link, options }
-}
```

Comment:
These exports, shouldn't we be re-exporting the equivalent bindings from `@makeswift/prop-controllers` so that we're not breaking things?